### DISCUSSION TITLE
DNS Pointing

### DISCUSSION THREAD

**Discussion by AienTech** [staticdeploy/staticdeploy#77] [Q&A]
Hey there,

it's more than a question rather than a issue, regarding the setup of the DNS (Cloudflare) with the project. Basically I have setup the staticdeploy on my own domain, with

```
A.   STATIC.MY_DOMAIN.TLD    123.456.789.000
```

the project itself is setup with docker (compose) and is proxied over a caddy instance.

The problem I'm facing at the moment is that, that when deploying a site, I configure its entrypoint like this:

```
CNAME    SITE.MY_DOMAIN.TLD    STATIC.MY_DOMAIN.TLD
```

but the domain is not reachable and won't be opened to show the website. This problem is unfortunately not documented, so I thought about asking it here. An answer would be much appreciated

**Comment by pscanf** [comment]
Hey @AienTech , happy to help. I need a few more details though.

> the domain is not reachable

What does this mean in practice? Do you get an error page by StaticDeploy? For instance:
![image](https://user-images.githubusercontent.com/3725458/138549133-7ba6e31c-c9c6-4027-86ac-06fa211326b0.png)

Or do you get something else entirely?

In general, I would check that:

- you have created an entrypoint having `SITE.MY_DOMAIN.TLD/` as `urlMatcher`
- the proxying is wired up correctly, so Caddy does indeed proxy requests to the StaticDeploy container
- Caddy forwards the `Host` header to the StaticDeploy container (it's used by StaticDeploy to figure out which entrypoint it needs to serve)
- Cloudflare forwards the `Host` header to Caddy

**Comment by AienTech** [comment]
Hey Paolo,

thanks for the answer, the issue I face is this one:

<img width="1363" alt="Screenshot 2021-10-23 at 10 41 49" src="https://user-images.githubusercontent.com/2676700/138549534-5c52b1b8-a57d-4975-a07b-db2cd7b1ef19.png">

so basically no errors from the staticdeploy instance.

regarding your check points:
- yes, that's exactly the case
- this might be an issue here, I might have missed something
- it would be effected by the second point
- It definitely does
